articles/integrations/aws-api-gateway/custom-authorizers/part-2.md

```diff
@@ -1,262 +1,143 @@
 ---
-title: AWS API Gateway Tutorial - Secure the API
 description: Step 2 of Amazon API Gateway Tutorial
 toc: true
 ---
 
-# AWS API Gateway Tutorial, Part 2: Create the Custom Authorizers
+# AWS API Gateway Tutorial, Part 2: Import and Deploy the API Gateway API
 
-In [step 1](/integrations/aws-api-gateway/custom-authorizers/part-1) of this tutorial, we showed you how to import, test, and deploy an API using Amazon Web Services' (AWS) API Gateway. In this tutorial, we will show you how to secure this API so that only those with the appropriate authorization may access the back-end behind the API.
+::: warning
+This portion of the tutorial has been adapted from the [official AWS example](http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-from-example.html). Please refer to this example for in-depth notes and discussion.
+:::
 
-To do this, we will be using API Gateway's custom [request] authorizers, which allow you to authorize your APIs using bearer token authorization strategies, such as OAuth 2.0 or SAML. For each incoming request, the following happens:
+In Part 1 of the AWS API Gateway tutorial, we will show you how to import and manage an API using API Gateway. More specifically, we will:
```

Comment:
this is part 2 yet it says `In Part 1 of the AWS API Gateway tutorial, we will show you how to`